packages/wp-now/src/main.ts

```diff
@@ -2,13 +2,12 @@ import { runCli } from './run-cli';
 
 const requiredMajorVersion = 18;
 
-const currentNodeVersion = parseInt(process.versions.node.split('.')[0]);
+const currentNodeVersion = parseInt(process.versions?.node?.split('.')?.[0]);
 
 if (currentNodeVersion < requiredMajorVersion) {
-	console.error(
+	console.warn(
 		`You are running Node.js version ${currentNodeVersion}, but this application requires at least Node.js ${requiredMajorVersion}. Please upgrade your Node.js version.`
```

Comment:
```suggestion
		`You are running Node.js version ${currentNodeVersion}, but this application recommends at least Node.js ${requiredMajorVersion} and isn't guaranteed to work on lower versions. Please upgrade your Node.js version.`
```

Reply:
Thank you! Updated: https://github.com/WordPress/playground-tools/pull/37/commits/cbe9b808d62bc4087815404bf54d35949478ebb8